coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
+  "result": {
+    "covered_percent": 94.15
+  }
+}
```

Comment:
That's very weird. Would you mind running

`rm coverage/.last_run.json`
and 
`rm .resulttest.json`